tests/integration/trust-registry-module.test.js

```diff
@@ -1,27 +1,27 @@
-import { randomAsHex } from '@polkadot/util-crypto';
```

Comment:
not sure why linting settings are off here, but all these could be single quotes not double to align with everywhere else